x/hub-genesis/types/params.go

```diff
@@ -1,31 +1,23 @@
 package types
```

Comment:
as above, we should be able to remove this file and all `Params` related stuff

Reply:
Yes you're right it should be removed
but is it ok if I do it in a followup PR? This PR is currently all tested etc
https://github.com/dymensionxyz/dymension-rdk/issues/452
I have made an issue for myself
Thanks